raw-data/PH/phoible_inventories.tsv

```diff
@@ -15409,27 +15409,27 @@
 				"ʃ"				
 				"x"				
 				"h"				
-				"sʼ"				
-				"ʃʼ"				
-				"xʼ"				
+				"sʼ"	"""Subsequent literature (by Wash 2001) suggests that glottalized means ejective for Beeler based on Beeler 1976 mentioning a 'staccato effect'."""			
+				"ʃʼ"	"""Subsequent literature (by Wash 2001) suggests that glottalized means ejective for Beeler based on Beeler 1976 mentioning a 'staccato effect'."""			
+				"xʼ"	"""Subsequent literature (by Wash 2001) suggests that glottalized means ejective for Beeler based on Beeler 1976 mentioning a 'staccato effect'."""			
```

Comment:
Having looked back at the sources, I think we ought to consider x and xʼ as uvular, not velar.  They are aligned in the same column as q, not k.  This is true in inventory 862 also: x is in the q column, not the k column.